tests/verbose_output.yml

```diff
@@ -0,0 +1,35 @@
+name: testsuite run in verbose mode
+testcases:
+- name: testsuite run in verbose mode
+  steps:
+  # spawn a venom sub-process and expect it to fail and make assertions on its error messages
+  # ensure no color to avoid annoying checks
+  - type: exec
+    script: NO_COLOR=1 {{.venom.executable}} run failing/verbose_output.yml {{.value.opt}}
+    range:
+      verbose:
+        opt: "-v"
+        op: Should
+      default:
+        opt: ""
+        op: ShouldNot
+    assertions:
```

Comment:
this test is kind of hacky, but basically it spawns another venom instance (with text colors disable) and run it once in verbose mode and once in non-verbose

then the assertions check that verbose output matches what is intended, wile the non-verbose does not display them

Reply:
ok